editor.planx.uk/src/@planx/components/List/model.ts

```diff
@@ -81,7 +99,7 @@ export interface Schema {
   max?: number;
 }
 
-export type UserResponse = Record<Field["data"]["fn"], string | string[]>;
+export type UserResponse = Record<Field["data"]["fn"], string | any[]>; // string | string[] | Feature[]
```

Comment:
`Feature[]` threw some type errors that'd be nice to come back, debug & tighten later ! Just defaulting to `any[]` for this first go